### ISSUE TITLE
Block handling below Y 0 (1.18)

### ISSUE THREAD

**Issue by Craftifyeu** [SkriptLang/Skript#4324]
### Skript/Server Version

Server Version: git-Tuinity-"4c31ea1" (MC: 1.17.1)
Skript Version: 2.6-beta2

Installed Skript Addons:
Skellett v1.9.11
skript-yaml v1.3.2
CreatioLib v0.1.02-alpha
skRayFall v1.9.25
Ticker v1.0.0
SharpSK v1.6.4
SkUniversal v2.12.5
DiSky v2.1
skDragon v0.17
skript-reflect v2.2.3
skript-db v0.2.0
Reqn v1.2.1
SkQuery v4.1.4
SkriptJSON v1.0.0
Ersatz v1.0
RandomSK v2.7.1
WolvSK v1.7.2
SkStuff v1.6.4.1

### Bug Description

Loop-block works above Y location 0 but when I go lower and I want the loop-block to destroy it, nothing happens, it can't detect both players and blocks below location zero

### Expected Behavior

Not working under location 0 (C&C datapack)

### Steps to Reproduce

bug description

### Errors or Screenshots

YouTube video [https://youtu.be/P6m9dk8XkX8](https://youtu.be/P6m9dk8XkX8)

### Agreement

- [X] I have read the guidelines above and confirm I am following them with this report.

**Comment by Craftifyeu** [comment]
sorry for my english (i'am using translator, i'am from the Czech Republic) :D

**Comment by bluelhf** [comment]
Sounds like Spigot being unprepared for 1.18. I don't think Skript can do much here

**Comment by Craftifyeu** [comment]
I don't know why, but it seems to me more like a community than a spigot. Why would other plugins work under location 0 and the script would not?

**Comment by AyhamAl-Ali** [comment]
Not sure if this is what you mean but I can't replicate this
```v
on chat:
	#broadcast "%blocks below player%"
	loop blocks below player:
		send "%loop-block%"
		break loop-block naturally
```

![image](https://user-images.githubusercontent.com/20037329/132507067-c81ee2ec-c624-4652-90c1-04ce45cba36c.png)
![image](https://user-images.githubusercontent.com/20037329/132507128-357d7afa-f593-4a7e-8111-f89be14469eb.png)

No addons in my server, skript 2.6-beta1 and paper 1.16.5
Try this without any of your addons

**Comment by Craftifyeu** [comment]
When I update to beta 3, 80% of my scripts stopped working.

**Comment by AyhamAl-Ali** [comment]
> When I update to beta 3, 80% of my scripts stopped working.

There were little breaking changes in 2.6 such as `singular quotes instead of double quotes in strings` and `addons patches` probably only those, if you have any script errors you can simply post them in [SkUnity discord](https://discord.gg/5FHkctwG) help channels

**Comment by TPGamesNL** [comment]
> When I update to beta 3, 80% of my scripts stopped working.

@Craftifyeu Do you get any errors (when reloading or in console)? Can you show a simple script that stopped working?

**Comment by TPGamesNL** [comment]
As 1.18 may have blocks below y=0, Skript should be updated to allow this
https://github.com/SkriptLang/Skript/blob/13dd0f120a8fd1d4408280e59377361c818cf297/src/main/java/ch/njol/skript/util/AABB.java#L67

@Craftifyeu I'm just curious: how do you have blocks at Y -1 in Minecraft 1.17?

**Comment by Craftifyeu** [comment]
> 1.17

Datapack C&C

**Comment by Craftifyeu** [comment]
> As 1.18 may have blocks below y=0, Skript should be updated to allow this
> https://github.com/SkriptLang/Skript/blob/13dd0f120a8fd1d4408280e59377361c818cf297/src/main/java/ch/njol/skript/util/AABB.java#L67
> 
> @Craftifyeu I'm just curious: how do you have blocks at Y -1 in Minecraft 1.17?

and it will be added before 1.18 or it would somehow work with the datapack to 1.17

**Comment by Craftifyeu** [comment]
> > When I update to beta 3, 80% of my scripts stopped working.
> 
> @Craftifyeu Do you get any errors (when reloading or in console)? Can you show a simple script that stopped working?

i rewrote my whole code, now my code's working in latest update.

**Comment by TPGamesNL** [comment]
> and it will be added before 1.18 or must wait for 1.18

I don't know, but it's an easy fix so it'll probably be fixed before 1.18 (although I don't know when 1.18 comes out)
